src/__tests__/mocks/configMock.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import { ReportPortalConfig } from '../../models';
+
+export const config: ReportPortalConfig = {
+  token: '00000000-0000-0000-0000-000000000000',
+  endpoint: 'https://reportportal.server/api/v1',
+  project: 'ProjectName',
+  launch: 'LaunchName',
+  description: 'Launch description',
+  attributes: [],
+};
```

Comment:
What the reason create this file? You don't pass it to the constructor(you read config inside constructor from json)

Reply:
In the case of tests, we pass it to the constructor. If we don't pass config, we get error, when run tests. 
`TypeError: Cannot read property 'debug' of undefined`

Reply:
Yes, because it depends on your implementation. Passing config through the constructor is workaround. But if you don't using it in real world, you shouldn't create this workaround for testing case.
It looks like you testing different flow.